src/test/java/picard/util/MiscTest.java

```diff
@@ -0,0 +1,26 @@
+package picard.util;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+
+public class MiscTest {
+
+    @Test
+    public void canWriteToDevNull() throws IOException {
+        File f = new File("/dev/null");
```

Comment:
If you really want to do this, it would be simpler to write
```Files.write(Paths.get("/dev/null"), "Just a test".getBytes());```

Reply:
not new code.